code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -16,7 +17,7 @@
 	. += /obj/item/organ/internal/xenos/plasmavessel/hunter
 
 /mob/living/carbon/alien/humanoid/hunter/movement_delay()
-	. = -1		//hunters are sanic
+	. = alien_movement_delay		//hunters are sanic
 	. += ..()	//but they still need to slow down on stun
```

Comment:
this should be on the base `/mob/living/carbon/alien/humanoid` level. no need to have the same copy paste code on every alien caste

Reply:
Addressed